### PR TITLE
DDPB-2572: Minimal infrastructure test suite

### PR DESCRIPTION
**Pull request by gregtyler**
Adds a new test suite for infrastructure tests, allowing quick smoke tests when making infra changes

### CODE REVIEW

File: tests/behat/features/deputy/07-infra-checks/00-documents.feature [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+  @infra
+  Scenario: Can upload documents
+    Given I am logged in as "laydeputy103@publicguardian.gov.uk" with password "Abcd1234"
+    When I set the report start date to "1/1/2016"
+    And I set the report end date to "31/12/2016"
+    And I click on "report-start, edit-documents, start"
+    And I fill in "document_wishToProvideDocumentation_0" with "yes"
+    And I click on "save-and-continue"
+    And I attach the file "empty-file.pdf" to "report_document_upload_file"
```
Comment (slizzio) [inline tests/behat/features/deputy/07-infra-checks/00-documents.feature]: Should this be testing the file checker for an error?

File: docker/confd/templates/behat.yml.tmpl [hunk 2]
```diff
@@ -73,6 +73,18 @@ default:
...
+        infra:
+            description: Minimal test suite to check infrastructure
+            paths: [ %paths.base%/features ]
+            filters:
+                tags: "@infra"
+            contexts:
+            {{ if exists "/api/database/name" }}
+                - DigidepsBehat\FeatureContext: [{ dbName: {{ getv "/api/database/name" }} }]
+            {{ else }}
+                - DigidepsBehat\FeatureContext: [{ dbName: "api" }]
```
Comment (slizzio) [inline docker/confd/templates/behat.yml.tmpl]: Small concern that if the env variable is not set then would this run tests against the wrong database?

Comment (gregtyler) [reply docker/confd/templates/behat.yml.tmpl]: I believe running against `api` would generally be the intended behaviour. It's how all the other suites are set up.

File: tests/behat/features/deputy/07-infra-checks/00-documents.feature (hunk 1 above)
Comment (gregtyler) [reply tests/behat/features/deputy/07-infra-checks/00-documents.feature]: Good point, added a check now

**Pull request merged**